server.go

```diff
@@ -170,6 +170,9 @@ func (s *Server) handleConn(conn net.Conn) {
 
 				if len(topic_names) > 0 {
 					writeErr = writer.WriteObjects(topic_names...)
+				} else {
+					// we need to return empty array here
+					_, writeErr = writer.Write([]byte{'*', '0', '\r', '\n'})
```

Comment:
As a sidenote, we can just do a `WriteObjects()` when https://github.com/secmask/go-redisproto/pull/2 is merged.